trtools/utils/utils.py

```diff
@@ -18,7 +18,9 @@
 
 def LoadSingleReader(
         vcf_loc: str,
-        checkgz: bool = True) -> Optional[cyvcf2.VCF]:
+        checkgz: bool = True,
+        lazy: bool = False,
+        samples: List[str] = None) -> Optional[cyvcf2.VCF]:
```

Comment:
Great suggestion! I have had similar hours for something else spent debugging after assuming sample order stays the same... I changed based on your suggestions.

(Here I just added it to avoid having cyvcf2 avoid loading any samples from the ref panel to save time, since we just need the INFO fields. but definitely this option could lead to problems down the road and set seems more appropriate)